test/mocha.spec.js

```diff
@@ -15,8 +15,8 @@ describe('mocha', () => {
 
   eyes.it('should allow overriding default timeout', function () {
     expect(this.timeout()).to.equal(60000);
-    this.timeout(500);
-    expect(this.timeout()).to.equal(500);
+    this.timeout(5000);
```

Comment:
why is the timeout increased?

Reply:
Great question.
I think what happened is that the snapshot was taken and sent to the applitools servers.
and it took more than 500ms to finish and send back the result.
So the test failed.
I wasn't really sure how else to approach this.